policy/release_manager.md

```diff
@@ -0,0 +1,49 @@
+# Release manager role
+
+## Appointment
+
+A release manager (RM) is appointed for a specific upcoming release.
+
+A RM may and should step down at any time, when they cannot fulfill the associated obligations. In this case, a new RM is appointed.
+
+The appointment of an RM ends with the declaration of the respective release's end-of-life by the RM.
```

Comment:
I was initially confused by what *end-of-life* means, but it was probably due to my initial misconception that RM is related to release *preparation*, and not preparation and subsequent maintenance of the MINOR version.

On re-read, I think it should have been obvious to me that release = release life cycle, not the moment at which it is released; and in any case the rest of the document clarifies it sufficiently. So I'd keep it as is. But if someone else thinks it's not immediately clear, then I'd suggest adding a preamble statement (before appointment section) like "the RM oversees the release life cycle, including preparation and subsequent maintenance".